static/js/redux/ui/Calendar.tsx

```diff
@@ -174,54 +176,40 @@ const Calendar = (props: CalendarProps) => {
 
   const addSISButton = props.registrarSupported ? (
     <div className="cal-btn-wrapper">
-      <button
-        type="submit"
-        form="form1"
-        className="save-timetable add-button"
-        data-for="sis-btn-tooltip"
-        data-tip
-        onClick={sisButtonClicked}
-      >
-        <img src="/static/img/addtosis.png" alt="SIS" style={{ marginTop: "2px" }} />
-      </button>
-      <ReactTooltip
-        id="sis-btn-tooltip"
-        class="tooltip"
-        type="dark"
-        place="bottom"
-        effect="solid"
-      >
-        <span>SIS Add to Cart</span>
-      </ReactTooltip>
+      <Tooltip title={<Typography fontSize={12}>SIS Add to Cart</Typography>}>
```

Comment:
Replacing react-tooltip with MUI Tooltip to ensure consistency (since I needed to use MUI Tooltip for the weekends switch)